CHANGELOG

```diff
@@ -2,6 +2,23 @@
 
 Release notes for the puppetlabs-puppetdb module.
 
+#### 5.1.0 - 2016/02/09
+
+This is a minor feature release.
+
+Detailed changes:
+
+* Use 'puppetdb-terminus' as the terminus package on RHEL, to avoid packaging
+  conflicts that could occur on upgrades from 2.x to 3.x. When
+  'puppetdb-terminus' is used on RHEL, the 'puppetdb-termini' package will be
+  installed as needed.
```

Comment:
The `puppetdb-terminus` version 3.x package on RHEL will install `puppetdb-termini` as a dependency.

?